src/plugins/orangehrmClaimPlugin/Dao/ClaimDao.php

```diff
@@ -482,6 +482,13 @@ protected function getClaimRequestPaginator(ClaimRequestSearchFilterParams $myCl
                 ->setParameter('empNumbers', $myClaimRequestSearchFilterParams->getEmpNumbers());
         }
 
+        if (!is_null($myClaimRequestSearchFilterParams->getEmployeeName())) {
```

Comment:
@ManulMax please check this

Reply:
This one need to fix. Checked in leave, recruitment and timesheet employee search. Those showing invalid message if we try to search random word.